cloud/api-gateway/src/main/java/com/qpay/apigateway/exception/AuthenticationException.java

```diff
@@ -0,0 +1,11 @@
+package com.qpay.apigateway.exception;
+
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.ResponseStatus;
+
+@ResponseStatus(code = HttpStatus.FORBIDDEN)
+public class AuthenticationException extends RuntimeException {
```

Comment:
You don't check login and password, you check jwt. 
```suggestion
public class AuthorizationException extends RuntimeException {
```